CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [3.0.3](https://github.com/aurelia/cli/compare/v3.0.2...v3.0.3) (2024-04-17)
```

Comment:
Please remove the change. When we release new version, the script will automatically update change log file.

Reply:
Done